doc/output-details-and-examples/src/output-files/eplustbl-lt-ext-gt.tex

```diff
@@ -1854,59 +1854,82 @@ \subsection{Component Sizing Summary}\label{component-sizing-summary}
 
 \subsection{Outdoor Air Summary}\label{outdoor-air-summary}
 
-The Outdoor Air Summary provides information for each zone on the average and minimum ventilation provided.
-
-The reports described so far in this section are displayed when specified in the Output:Table:SummaryReports object. They are either on or off and are not customizable. The next few types of tabular reports are customizable. You can specify the report variables being used in each one.
-
-Directly following is an example of the report. The key used to obtain this report is OutdoorAirSummary.
+The Outdoor Air Summary provides information for each zone on the average and minimum ventilation provided. Directly following is an example of the report. The key used to obtain this report is OutdoorAirSummary.
 
```

Comment:
Do we need to include other possible keys for this table?

    Output:Table:SummaryReports,
      AllSummary;              !- Report 1 Name